components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -361,8 +358,8 @@ - (void)setFloatingButtonHidden:(BOOL)floatingButtonHidden animated:(BOOL)animat
     }];
   } else {
     _floatingButton.hidden = NO;
-    [self cutBottomAppBarViewAnimated:animated];
     [_floatingButton expand:animated completion:nil];
+    [self cutBottomAppBarViewAnimated:animated];
```

Comment:
Why did we need to change the order here?

Reply:
It doesn't affect the behavior. Simple because I saw `healBottomAppBarViewAnimated` was called before `[_floatingButton collapse]` on the healing part. So in the cutting part, I thought it should expand first as the reverse way. The behavior doesn't change.

Reply:
If we can revert this change without affecting the PR let's do so in the spirit of minimizing deltas here.

Reply:
Sure. Reverted in 4961552806e27310659c898640fd7e64822f6e29. Will merge after CI is finished.